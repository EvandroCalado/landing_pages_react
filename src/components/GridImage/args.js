export const args = {
  background: false,
  title: 'My grid',
  description:
    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.',
  grid: [
    {
      altText: 'Teste 1',
      image:'https://source.unsplash.com/random/800x800?r=1',
    },
    {
      altText: 'Teste 2',
      image:'https://source.unsplash.com/random/800x800?r=2',
    },
    {
      altText: 'Teste 3',
      image:'https://source.unsplash.com/random/800x800?r=3',
    },
    {
      altText: 'Teste 4',
      image:'https://source.unsplash.com/random/800x800?r=4',
    },
    {
      altText: 'Teste 5',
      image:'https://source.unsplash.com/random/800x800?r=5',
    },
    {
      altText: 'Teste 6',
      image:'https://source.unsplash.com/random/800x800?r=6',
    },
  ],
};
