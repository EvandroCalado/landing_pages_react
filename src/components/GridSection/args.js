export const args = {
  background: false,
  title: 'My grid',
  description:
    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.',
  grid: [
    {
      title: 'Teste 1',
      description:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.',
    },
    {
      title: 'Teste 2',
      description:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.',
    },
    {
      title: 'Teste 3',
      description:
        'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.',
    },
  ],
};
