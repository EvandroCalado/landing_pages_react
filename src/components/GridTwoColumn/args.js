export const args = {
  title: 'Grid Two Column',
  text: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur, omnis qui cumque reiciendis accusantium reprehenderit optio voluptates incidunt. Fugit facilis debitis officia repellat ab laudantium voluptatibus incidunt expedita a nihil!`,
  image: "assets/images/javascript.svg"
}