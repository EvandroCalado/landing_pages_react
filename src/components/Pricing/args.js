export const args = {
  id: 1,
  __component: 'section.pricing',
  description_one:
    'The release of Apple Silicon-based Macs at the end of last year generated a flurry of news coverage and some surprises at the machine’s performance. This post details some background information on the experience of porting Firefox to run natively on these CPUs.',
  description_two:
    'We’ll start with some background on the Mac transition and give an overview of Firefox internals that needed to know about the new architecture, before moving on to the concept of Universal Binaries.',
  title: 'pricing',
  table: [
    {
      id: 1,
      head: 'title 1',
      body: 'content1',
      footer: 'footer 1',
    },
    {
      id: 2,
      head: 'title 2',
      body: 'content2',
      footer: 'footer 2',
    },
    {
      id: 3,
      head: 'title 3',
      body: 'content3',
      footer: 'footer 3',
    },
    {
      id: 4,
      head: 'title 4',
      body: 'content4',
      footer: 'footer 4',
    },
    {
      id: 5,
      head: 'title 5',
      body: 'content5',
      footer: 'footer 5',
    },
  ],
  matadata: {
    id: 7,
    name: 'pricing',
    section_id: '#pricing',
    background: false,
  },
};
