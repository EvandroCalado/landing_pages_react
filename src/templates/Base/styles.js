import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    margin-top: 5.4rem;

    @media ${theme.media.medium} {
      margin-top: 0;
    }
  `}
`;
