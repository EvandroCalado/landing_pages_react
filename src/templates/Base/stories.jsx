import Base from '.';
import args from './mock';

const BaseStories = {
  title: 'Templates/Base',
  component: Base,
  args: args,
};

export const Template = (args) => {
  return (
    <div>
      <Base {...args} />
    </div>
  );
};

export default BaseStories;
