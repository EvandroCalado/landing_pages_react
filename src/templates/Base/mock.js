import { mock } from '../../components/Navbar/mock';
import { args as GridMock } from '../../components/GridSection/args';
import GridSection from '../../components/GridSection';

const args = {
  children: (
    <>
      <GridSection {...GridMock} background />
      <GridSection {...GridMock} />
      <GridSection {...GridMock} background />
      <GridSection {...GridMock} />
      <GridSection {...GridMock} background />
      <GridSection {...GridMock} />
    </>
  ),
  links: mock,
  logoData: {
    text: 'logo',
    link: '#',
  },
  html: '<p>Teste de footer</p>',
};

export default args;
