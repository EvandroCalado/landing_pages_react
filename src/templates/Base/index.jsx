import P from 'prop-types';
import * as Styled from './styles';
import Menu from '../../components/Menu';
import Footer from '../../components/Footer';
import GoTop from '../../components/GoTop';

const Base = ({ links, logoData, html, children }) => {
  return (
    <>
      <Menu links={links} logoData={logoData} />
      <Styled.Container>
        {children}
        <Footer footerHtml={html} />
      </Styled.Container>
      <GoTop />
    </>
  );
};

Base.propTypes = {
  children: P.node.isRequired,
  ...Menu.propTypes,
  html: P.string.isRequired,
};

export default Base;
