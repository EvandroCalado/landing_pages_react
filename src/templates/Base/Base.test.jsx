import Base from '.';
import { renderTheme } from '../../styles/render.theme';
import args from './mock';

describe('<Base />', () => {
  it('should render', () => {
    const { container } = renderTheme(<Base {...args} />);
    expect(container).toMatchSnapshot();
  });
});
