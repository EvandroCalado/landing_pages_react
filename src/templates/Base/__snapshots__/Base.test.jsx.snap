// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base /> should render 1`] = `
.c11 {
  margin-top: 5.4rem;
}

.c4 {
  width: 100%;
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c7 {
  color: #0a1128;
  font-size: 2.4rem;
  font-weight: 900;
  text-transform: uppercase;
  text-align: center;
}

.c14 {
  color: #fff;
  font-size: 6.4rem;
  font-weight: 900;
  text-transform: uppercase;
  text-align: center;
}

.c18 {
  color: #fff;
  font-size: 3.2rem;
  font-weight: 900;
  text-transform: none;
  text-align: center;
}

.c22 {
  color: #0a1128;
  font-size: 6.4rem;
  font-weight: 900;
  text-transform: uppercase;
  text-align: center;
}

.c23 {
  color: #0a1128;
  font-size: 3.2rem;
  font-weight: 900;
  text-transform: none;
  text-align: center;
}

.c2 {
  position: fixed;
  z-index: 5;
  top: 0;
  left: 0;
  right: 0;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
  border-bottom: #ddd;
  background-color: #fff;
}

.c2 > .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c2 .c6 {
  margin-top: 0;
  margin-bottom: 0;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 {
  display: none;
  width: 4rem;
  height: 4rem;
  position: fixed;
  z-index: 6;
  top: 2rem;
  right: 2rem;
  border: none;
  background-color: #0a1128;
  color: #fff;
  pointer-events: all;
}

.c0 > svg {
  width: 2.5rem;
  height: 2.5rem;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  padding: 2rem;
}

.c8 > img {
  height: 3rem;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c10 {
  font-size: 1.6rem;
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  padding: 1.6rem;
  color: #0a1128;
  position: relative;
}

.c10::after {
  content: "";
  position: absolute;
  bottom: 0.75rem;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background-color: #dc143c;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c10:hover::after {
  width: 50%;
  left: 25%;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c20 {
  font-size: 2.4rem;
}

.c24 {
  text-align: center;
  border-top: 1px solid #ddd;
}

.c24 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c24 .c19 {
  font-size: 1.6rem;
}

.c25 {
  background-color: #fff;
  color: #0a1128;
  border: 1px solid #0a1128;
  border-radius: 50%;
  position: fixed;
  bottom: 2rem;
  right: 2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 5rem;
  height: 5rem;
  z-index: 6;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c13 .c19 {
  margin-bottom: 6.4rem;
}

.c15 {
  margin-bottom: 6.4rem;
  text-align: center;
}

.c16 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
  overflow: hidden;
  width: 100%;
}

.c17 .c6 {
  position: relative;
  left: 5rem;
  text-align: left;
}

.c17 .c6::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  font-size: 7rem;
  top: -3.5rem;
  left: -5rem;
  -webkit-transform: rotate(15deg);
  -ms-transform: rotate(15deg);
  transform: rotate(15deg);
}

.c12 {
  background-color: #fff;
  color: #0a1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #0a1128;
  color: #fff;
}

.c21 {
  background-color: #fff;
  color: #0a1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (max-width:768px) {
  .c11 {
    margin-top: 0;
  }
}

@media (max-width:768px) {
  .c14 {
    font-size: 4rem;
  }
}

@media (max-width:768px) {
  .c22 {
    font-size: 4rem;
  }
}

@media (max-width:768px) {
  .c2 {
    height: 100vh;
    visibility: hidden;
    opacity: 0;
  }

  .c2 > .c3 {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 1fr;
    height: 100vh;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    overflow-y: auto;
  }

  .c2 .c6 {
    padding-bottom: 3.2rem;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c5 {
    display: block;
    text-align: center;
    padding: 4.8rem 0;
  }
}

@media (max-width:768px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c9 {
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
  }
}

@media (max-width:768px) {
  .c16 {
    grid-template-columns: 1fr;
  }
}

<div>
  <button
    aria-label="Open/Close menu"
    class="c0"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    class="c2"
  >
    <div
      class="c3 c4"
    >
      <div
        class="c5"
      >
        <h1
          class="c6 c7"
        >
          <a
            class="c8"
            href="#"
          >
            logo
          </a>
        </h1>
        <nav
          aria-label="Main menu"
          class="c9"
        >
          <a
            class="c10"
            href="#target 1"
            target="_self"
          >
            link 1
          </a>
          <a
            class="c10"
            href="#target 2"
            target="_self"
          >
            link 2
          </a>
          <a
            class="c10"
            href="#target 3"
            target="_self"
          >
            link 3
          </a>
          <a
            class="c10"
            href="#target 4"
            target="_self"
          >
            link 4
          </a>
          <a
            class="c10"
            href="#target 5"
            target="_self"
          >
            link 5
          </a>
          <a
            class="c10"
            href="#target 6"
            target="_self"
          >
            link 6
          </a>
          <a
            class="c10"
            href="#target 7"
            target="_self"
          >
            link 7
          </a>
          <a
            class="c10"
            href="#target 8"
            target="_self"
          >
            link 8
          </a>
          <a
            class="c10"
            href="#target 9"
            target="_self"
          >
            link 9
          </a>
        </nav>
      </div>
    </div>
  </div>
  <div
    class="c11"
  >
    <div
      class="c12"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <h2
            class="c15"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </h2>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c21"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c22"
          >
            My grid
          </h2>
          <h2
            class="c15"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </h2>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 1
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 2
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 3
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <h2
            class="c15"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </h2>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c21"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c22"
          >
            My grid
          </h2>
          <h2
            class="c15"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </h2>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 1
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 2
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 3
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <h2
            class="c15"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </h2>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 1
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 2
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c18"
              >
                Teste 3
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c21"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c22"
          >
            My grid
          </h2>
          <h2
            class="c15"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.
          </h2>
          <div
            class="c16"
          >
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 1
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 2
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
            <div
              class="c17"
            >
              <h3
                class="c6 c23"
              >
                Teste 3
              </h3>
              <div
                class="c19 c20"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="c24"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c19 c20"
        >
          <p>
            Teste de footer
          </p>
        </div>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="c25"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M7.41 15.41 12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
